src/third_party/catch/include/helpers.hpp

```diff
@@ -253,14 +253,16 @@ MONGOCXX_INLINE_NAMESPACE_END
     auto concern_copy = libmongoc::write_concern_copy.create_instance(); \
     concern_copy->interpose([](const mongoc_write_concern_t*) { return nullptr; }).forever();
 
-#define MOCK_READ_PREFERENCE                                                     \
-    auto read_prefs_get_max_staleness_seconds =                                  \
-        libmongoc::read_prefs_get_max_staleness_seconds.create_instance();       \
-    auto read_prefs_get_mode = libmongoc::read_prefs_get_mode.create_instance(); \
-    auto read_prefs_get_tags = libmongoc::read_prefs_get_tags.create_instance(); \
-    auto read_prefs_set_max_staleness_seconds =                                  \
-        libmongoc::read_prefs_set_max_staleness_seconds.create_instance();       \
-    auto read_prefs_set_mode = libmongoc::read_prefs_set_mode.create_instance(); \
-    auto read_prefs_set_tags = libmongoc::read_prefs_set_tags.create_instance();
+#define MOCK_READ_PREFERENCE                                                       \
+    auto read_prefs_get_max_staleness_seconds =                                    \
+        libmongoc::read_prefs_get_max_staleness_seconds.create_instance();         \
+    auto read_prefs_get_mode = libmongoc::read_prefs_get_mode.create_instance();   \
+    auto read_prefs_get_tags = libmongoc::read_prefs_get_tags.create_instance();   \
+    auto read_prefs_set_max_staleness_seconds =                                    \
+        libmongoc::read_prefs_set_max_staleness_seconds.create_instance();         \
+    auto read_prefs_set_mode = libmongoc::read_prefs_set_mode.create_instance();   \
+    auto read_prefs_set_tags = libmongoc::read_prefs_set_tags.create_instance();   \
+    auto read_prefs_set_hedge = libmongoc::read_prefs_set_hedge.create_instance(); \
```

Comment:
slight nit, these are otherwise organized with all the getters first, then all the setters after, in what appears to be alphabetical order.  Might as well maintain the ordering.

Reply:
Done.